src/templates/service.mustache

```diff
@@ -21,35 +21,39 @@ export default class {{name}} {
   {{/description}}
   public static {{name}} = async (
     {{#parameters}}
-    {{#description}}
-    /**
-     * @description {{description}}
-     */
-    {{/description}}
-    {{name}}{{#isOption}}?{{/isOption}}: {{&type}},
+    {{>parameters}}
     {{/parameters}}
   ){{#responseType}}:Promise<{{{responseType}}}>{{/responseType}} => {
     const res = await _request{{#responseType}}<{{{responseType}}}>{{/responseType}}(`{{&url}}`, {
       method: '{{method}}',
-      {{#hasBody}}
+      {{#header.length}}
+      header: {
```

Comment:
headers is better for fetch api
see: [fetch api](https://developer.mozilla.org/en-US/docs/Web/API/fetch)